### PR TITLE
Adding Badges in README.md

### PR DESCRIPTION
**Pull request by akashprasher**
Added Badges for :

1. NPM Current
2. Typescript
3. JavaScript
4. License
5. Issues Count
6. Fork Count

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
   <img width="400" alt="proof" src="./packages/docs/src/media/proof.color.text.svg"/>
 </h1>
 
+<p align="center">
+  <a href="#"><img alt="npms" src="https://img.shields.io/npm/v/npm"></a>
```
Comment (adierkens) [inline README.md:6]: Thanks for taking on this work. Looking at the contents of the badges, I don't think this reflects the current version of the `@proof-ui` packages on npm. Do you know what this is pulling from?

Comment (akashprasher) [reply README.md]: Sorry about this, I just updated it with the current version of @proof-ui, which is v0.1.5.

**Comment by akashprasher** [comment]
Hey @adierkens ! Is it possible for you to add "hacktoberfest-accepted" label in this PR?

**Comment by adierkens** [comment]
:rocket: PR was released in [`v0.1.6`](https://github.com/intuit/proof/releases/tag/v0.1.6) :rocket:

**Pull request merged**